app/src/main/java/ch/epfl/polychef/users/User.java

```diff
@@ -24,16 +30,17 @@
  */
 public class User implements Serializable, Miniatures {
 
+    private static final String TAG = "User";
     private String email;
     private String username;
+    private String key;
     private int profilePictureId;
+    private Rating rating;
```

Comment:
maybe we should rename this attribute so we underline the difference with the Rating of a Recipe ?